BedrockPlugin.h

```diff
@@ -82,6 +87,9 @@ class BedrockPlugin {
     // Set to true if we don't want to log timeout alerts, and let the caller deal with it.
     virtual bool shouldSuppressTimeoutWarnings();
 
+    // Loads a given secure data object into the plugins local secure data map.
```

Comment:
Can you elaborate on what "secure data" means?  As someone coming into this cold, I can't figure out what it does and what makes it "secure".

Reply:
Nothing really makes it secure, honestly. This is for key loading, but we aren't doing anything special with regards to security around it.